### PR TITLE
Allow users to swipe screenshots in fullscreen

### PR DESCRIPTION
**Pull request by sverzegnassi**

### CODE REVIEW

File: openstore/Components/ScreenshotsCarousel.qml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2015 - Michael Zanetti <michael.zanetti@ubuntu.com>
+ * Copyright (C) 2017 - Michael Zanetti <verzegnassi.stefano@gmail.com>
```
Comment (bhdouglass) [inline openstore/Components/ScreenshotsCarousel.qml]: @sverzegnassi lol, copy-pasta

**Comment by bhdouglass** [comment]
@sverzegnassi this is nice! Although the close icon is hidden if you have a white-ish image (for example I was looking at Falcon's screenshots).

**Comment by sverzegnassi** [comment]
Fixed! :D

**Comment by bhdouglass** [comment]
I merged this via the cli, but github is being weird, so I'll just close this out.

**Pull request closed without merging**